### PR TITLE
Create kehan-yang.yaml

### PR DESCRIPTION
**Pull request by KehanGit**

### CODE REVIEW

File: kehan-yang.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  role: Computational Geoscientist
+  organizations:
+    - name: M3 Works LLC
+  bio: >
+    Kehan is a computational geoscientist at M3 Works LLC, specilizing in operational snow modeling in the Western U.S.
+    She has a backaground in snow hydrology, remote sensing, and machine learning. 
+    She is passionate about investigating methods that intergate satellite data, physically-based models, and machine learning tools to 
+    enhance our understanding of mountain snowpack and water avaialbility.
+     
+  expertise:
```
Comment (scottyhq) [inline kehan-yang.yaml]: @jomey I'm not sure why this isn't rendering in the preview. Maybe the script is sensitive to newlines?

**Comment by jomey** [comment]
@KehanGit - You need to place your profile into the `team` folder in order to have you added. The team-builder script only looks for added profiles there.

File: team/kehan-yang.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+- title: Kehan Yang
+  avatar: https://avatars.githubusercontent.com/u/32207757?v=4
+  role: Computational Geoscientist
+  organizations:
+    - name: M3 Works LLC
+  bio: Kehan is a computational geoscientist at M3 Works LLC, specilizing in operational snow modelings.
```
Comment (jomey) [inline team/kehan-yang.yaml]: Should have noticed that earlier, but multi-line strings in Yaml need to be started with a `>`
[Here an example from another team member](https://github.com/uwhackweek/event-page-2024/pull/34/files#diff-3d20d2961fbdb5fc302e636e66b85406bba688ed99dc7ec6b6656bded69b9274R6)

**Review by jomey — CHANGES_REQUESTED** [review]

File: team/kehan-yang.yaml (hunk 2 above)
Comment (KehanGit) [reply team/kehan-yang.yaml]: Thanks!

File: team/kehan-yang.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+- title: Kehan Yang
+  avatar: https://avatars.githubusercontent.com/u/32207757?v=4
+  role: Computational Geoscientist
+  organizations:
+    - name: M3 Works LLC
+  bio: >
```
Comment (jomey) [inline team/kehan-yang.yaml]: The YAML indentation rules remain the same when using the arrow:
```
bio: >
  Kehan is ..
```

**Review by jomey — APPROVED** [review]

**Comment by KehanGit** [comment]
Thanks @jomey

**Pull request merged**